tests/phpunit/includes/ContentParserTest.php

```diff
@@ -2,6 +2,8 @@
 
 namespace SMW\Tests;
 
+use MediaWiki\Content\Renderer\ContentRenderer;
```

Comment:
Since this was added in MW 1.38, would it be better to not `use` it, and rather have it in the docstring below? Not sure if it makes a difference for < MW 1.38 (at least, tests don't complain).

Reply:
`use` statements do not matter. They can reference symbols that do not exist. They only get used for lookup if you actually use the class during code execution. In other words: this is fine.png